__tests__/utils/locale-detector.test.ts

```diff
@@ -0,0 +1,28 @@
+import { matchOsLocaleToSupportedLocale } from "../../app/utils/locale-detector"
```

Comment:
side note: I have been confused as some test are "in line", ie in the same folder as the other one, and other tests are in `__tests__`.

I dpn't really care which way we go, but having consistency here would be useful. feel free to take this PR if you'd like probably a small one

Reply:
I will create a follow up PR to bring consistency to the testing.